frontend/workflows/projectSelector/src/hello-world.tsx

```diff
@@ -72,6 +73,8 @@ export const useReducerState = () => {
   return React.useContext(StateContext);
 };
 
+export const stateStore = new StateStore();
```

Comment:
don't think this should be exported, state will come from the hook.

Reply:
yea there are a few things I want to discuss in our sync that I'm not too sure about